packages/overdrive/lib/components/private/CheckableBase/CheckableBase.tsx

```diff
@@ -43,29 +43,58 @@ export const CheckableBase: FunctionComponent<Props> = ({
 	const nakedLabel = ['string', 'number'].includes(typeof label);
 
 	return (
-		<Box className={clsx(styles.root, styles.tappable, className)}>
-			<input
+		<Box
```

Comment:
See comments above about Radio/Checkbox think those issues are both in this file?